lib/cabin/mixins/logger.rb

```diff
@@ -3,6 +3,17 @@
 # This module implements methods that act somewhat like Ruby's Logger class
 # It is included in Cabin::Channel
 module Cabin::Mixins::Logger
+
+  def self.included(klass)
+    klass.condition do |event, subscription|
+      if subscription.nil?
+        true
+      else
+        LEVELS[(subscription.options[:level] || :debug)] >= LEVELS[event[:level]].to_i
```

Comment:
I wonder if we should wrap this condition within a method of the logger class? it looks to me like a common check, also would benefit other level verifications, right?. what do you think?